src/js/helpers/prepare.js

```diff
@@ -1,11 +1,22 @@
 /* Clearing variables */
 
 import calculateOffset from './calculateOffset';
+import getInlineOption from './getInlineOption';
 
 const prepare = function ($elements, options) {
   $elements.forEach((el, i) => {
+    const positionIn = calculateOffset(el.node, options.offset);
+    const positionOut = positionIn + window.innerHeight;
```

Comment:
Sorry for such a long time since last message. I was really busy.

Calling `.offset` results in a reflow. So I'd rather calculate this explicitly when `mirror` option is turned on, not every time.

Yes, I was thinking about:
```
(elementOffsetTop + el.offsetHeight + window.innerHeight) - additionalOffset
```

I'm willing to export two calculation options in `calculateOffset` file like -> `getPositionIn` and `getPositionOut`, so we can use it like this:
```
import { getPositionIn, getPositionOut } from './calculateOffset';
```
It would then make sense to rename `calculateOffset` to `offsetCalculator`.